scripts/ws-realm.sh

```
@@ -263,29 +298,46 @@ ws_overlay_list() {
+    # Block community realms from shadowing it via URL.
+    if [[ "$repo_name" == "realm-template" ]]; then
+        echo "ERROR: 'realm-template' is reserved for the upstream tutorial realm." >&2
+        echo "  Use 'ws realm init' to clone the template, or rename the URL's repo." >&2
+        exit 1
+    fi
+
+    local target="$REALMS_DIR/$repo_name"
     if [[ -d "$target" ]]; then
-        echo "ERROR: Live overlay already exists at $target." >&2
-        echo "  Remove it first or use 'ws overlay use' to switch." >&2
+        echo "ERROR: Realm '$basename' already exists at $target." >&2
```

Shellcheck (0.11.0)

[warning] basename is referenced but not assigned (for output from commands, use "$(basename ...)" ).

(SC2154)